source/educators/how-tos/content-tagging-how-tos/build_taxonomy_using_template.rst

```diff
@@ -0,0 +1,92 @@
+
+  For a tag like “United States” which is a “root” tag in the taxonomy, we leave “parent_id” empty. We can also leave “comments” empty.
+
+  **Choosing an ID:** The “id” column is required, but its exact value is not particularly important at this stage. The main requirement is that each row (each tag) has a unique ID. (Though remember that the “values” of each row also must be unique.) For learning purposes, feel free to use numbers like 1, 2, 3, 4 as the IDs. You could also just make the “id” the same as the “value” (i.e. put “United States” as both the ID and the value), or make up a short ID like “USA”. To understand more, and learn how IDs play an important role when updating the taxonomy, please read `Concept: Why does each tag need an ID when importing a taxonomy? <https://openedx.atlassian.net/l/cp/U1i001z1>`_.
+
+3. Creating "child" tags
+************************
+
+* Now, to add the “California” tag, we put it in the next row. Let’s give it the ID “2” and put the value as “California”. Now, because we want this tag to be a “child” of the “United States” tag in our taxonomy, we also need to set the **parent_id** field. In this case the parent is “United States”, which we gave the ID 1, so we’d put 1 as the parent_id, like this:
+
+  .. image:: /_images/educator_how_tos/ctag_taxonomy_template_childtag1.png
+     :alt: Populated spreadsheet with four columns labeled: id, value, parent_id, and comments.
+
+* Likewise, to create “Los Angeles” as a child tag of “California”, we need to give it an ID (say, 3), and set its parent ID to the ID we gave to “California” (in this case, 2).
+
+  .. image:: /_images/educator_how_tos/ctag_taxonomy_template_childtag2.png
+     :alt: Populated spreadsheet with four columns labeled: id, value, parent_id, and comments.
+
+  So far, this would result in the following taxonomy:
+
+  * United States
+
+    * California
+
+      * Los Angeles
+
+* Now, continue to add the rest of the tags. You can put notes in the “comments” column if you want; that column and any other columns besides “id”, “value”, and “parent_id” are ignored by the system during import.
+
+  .. image:: /_images/educator_how_tos/ctag_taxonomy_template_childtag3.png
+     :alt: Populated spreadsheet with four columns labeled: id, value, parent_id, and comments.
+
+* Save the spreadsheet using the default format (e.g. .xlsx), in case you want to edit it later.
+
+* Export the spreadsheet to a CSV file.
+
+  * In Excel, use File > Save As…, and set the “File format” to “CSV UTF-8 (comma-delimited) (.csv)” (preferred) or “Comma Separated Values (.csv)” then enter a filename and save it.
+  * In Google Sheets, use File > Download > Comma Separated Values (.csv).
+  * In Numbers, use File > Export To > CSV… and use the default options.
+
+* Import your taxonomy following the `How-to: Import and export a taxonomy guide <https://openedx.atlassian.net/l/cp/axcFGD1Q>`_.
```

Comment:
I'll make a note on this wiki page to update this How-To with a relative link

Reply:
@crathbun428 - Thanks. Looks good to me! Merge away.